robots/gantry.h

```diff
@@ -199,7 +215,8 @@ class Gantry
 private:
     BYTE m_BoardId;
     bool m_IsMovingLine = false;
-    static constexpr std::array<double, 3> PulsesPerMillimetre = { 7.49625, 8.19672, 13.15789 };
+    static constexpr Vector3<double> PulsesPerMillimetre = { 7.49625, 8.19672, 13.15789 };
```

Comment:
Could you maybe add a comment explaining the source of these magic numbers incase someone else ever has to assume your role as chief gantry-wrangler?

Reply:
Sure, I'll stick a comment in.

The source is Chris Johnson's Matlab script. I have to assume he measured these values empirically, but they do seem to be relatively accurate.